### DISCUSSION TITLE
Can we set out-width and out-height for latex figures?

### DISCUSSION THREAD

**Discussion by shabbychef** [quarto-dev/quarto-cli#2415] [Q&A]
It seems that the options
```
#| out-width: "0.5\\textwidth"
#| out-height: "0.32\\textwidth"
```
have no effect on .pdf output: the figure is not half sized; when I save the intermediate .tex file, there are no "textwidth" in the doc, _etc_. 
Is this expected? How should one set the output figure width in pdf output?

**Comment by cderv** [comment]
I noticed the same thing. This is a difference compare to using **knitr** in R Markdown.

`out-width` will be passed as `width` attributes. Example 
````markdown
![](test2_files/figure-pdf/unnamed-chunk-3-1.pdf){fig-pos='H' width=\\textwidth}
````

It seems pandoc does not support setting `\textwidth` passed in the `width` attributes. 🤔 

Though you can use `%` units which Pandoc's deal with specially
````yaml
#| out-width: "50%"
#| out-height: "32%"
````

````latex
\includegraphics[width=0.5\textwidth,height=0.32\textheight]{test2_files/figure-pdf/unnamed-chunk-3-1.pdf}
````

I think I'll open an issue about the latex string passed to width to see what we can do.

**Reply by cderv** [reply]
I opened #2417
